ios/browser/ui/webui/brave_web_ui_controller_factory.mm

```diff
@@ -39,30 +42,35 @@
   return std::make_unique<T>(web_ui, url);
 }
 
+WebUIIOSFactoryFunction GetUntrustedWebUIIOSFactoryFunction(const GURL& url) {
+  if (!url.SchemeIs(kChromeUIUntrustedScheme)) {
+    return nullptr;
+  }
```

Comment:
This seems redundant given we're already checking the scheme in the only place its called